provisioning/device/tests/ProvisioningTransportRegisterRequestTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.UnitTests
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class ProvisioningTransportRegisterRequestTests
+    {
+
+        private readonly string _globalDeviceEndpoint = "endpoint";
+        private readonly string _idScope = "id";
+
+        [TestMethod]
+        public void ProvisioningTransportRegisterRequest_DefaultPayload()
+        {
+            // arrange - act
+            var auth = new Mock<AuthenticationProvider>();
+            auth.Setup(p => p.GetRegistrationId()).Returns("registrationId");
```

Comment:
is the setup necessary for this test?

Reply:
Good catch, I tested it without and looks like it's not needed.